keep-ui/.gitignore

```diff
@@ -8,6 +8,8 @@ pids
 *.pid
 *.seed
 
+!lib
```

Comment:
What is it? Just curious.

Reply:
overriding global gitignore to include new file in /entities/incidents/lib/